kernels/volk/volk_16ic_deinterleave_16i_x2.h

```diff
@@ -274,7 +274,7 @@ static inline void volk_16ic_deinterleave_16i_x2_generic(int16_t* iBuffer,
 extern void volk_16ic_deinterleave_16i_x2_a_orc_impl(int16_t* iBuffer,
                                                      int16_t* qBuffer,
                                                      const lv_16sc_t* complexVector,
-                                                     unsigned int num_points);
+                                                     int num_points);
```

Comment:
The signature for the other implementations is `unsigned int num_points`. I vaguely remember an issue, where this discrepancy was fixed. Should we really re-introduce this mismatch?

Reply:
Here I've changed the `_orc_impl` functions, which are not protokernels themselves. (They're called by the `_orc` protokernels.) I don't think that should be a problem.

Reply:
The only mismatch will be the point where `_orc` calls `_orc_impl` and an explicit cast could be added there if needed. (Perhaps to silence compiler warnings?)

Reply:
It looks like warnings only show up if `-Wsign-conversion` is turned on. Doing show produces tons of warnings from other parts of VOLK. If you'd like, I could add explicit casts to the `_orc_impl` invocations to suppress the warnings there.

As far as I can see, a conversion from `unsigned int` to `int` is unavoidable because ORC uses `int` for its loop index.

Reply:
Thanks for the clarification. That'll work then.